### ISSUE TITLE
Free drawing is buggy on iPad: missing inputs and too sensitive to double-taps

### ISSUE THREAD

**Issue by steenfatt** [excalidraw/excalidraw#1935]
It's difficult to do actual handwriting on the iPad. The two main issues seem to be:

- sometimes missing some inputs

- too sensitive to double-tapping (wants to bring up the iPad keyboard)

See example video: https://photos.app.goo.gl/XRgwqB3pEu6QoQJP8

**Comment by steenfatt** [comment]
Pull request #1937 partly fixes this. Thanks for that!

The double-tapissue seems gone. However, the missing inputs are still an issue. In some sessions they only happen occasionally whereas in others it's really bad. In the latter case I'm only able to draw tiny segments before losing the input. This happened to me before but I didn't capture it, and the next day the problem was gone (well, reduced to only occasional input loss).

However just now when trying the double-tap fix, it started happening again: https://photos.app.goo.gl/3MjnrT2yXvgEtAss7

Resetting the canvas or even reloading the page does NOT fix it. It just randomly comes and goes!

**Comment by dwelle** [comment]
Is it related just to ipad pencil?

**Comment by steenfatt** [comment]
No, sorry, I should have mentioned that it's the exact same problem when using my finger!

**Comment by dwelle** [comment]
Interesting. I can't repro on my Android :(.
